apps/web/src/utils/channels.ts

```diff
@@ -25,6 +25,14 @@ interface IChannelDefinition {
   type: NodeTypeEnum;
 }
 
+export const CHANNEL_TYPE_TO_STRING: Record<ChannelTypeEnum, string> = {
```

Comment:
is it possible to use this one inside of `WorkflowNode` as well so we have this mapping only once?

Reply:
yes, just need to lower-case it in the `WorkflowNode`